modules/olm-status-viewing-cli.adoc

```diff
@@ -17,18 +17,21 @@ You can view Operator Subscription status using the CLI.
 
 . List Operator Subscriptions:
 +
+[source,terminal]
 ----
 $  oc get subs -n <operator_namespace>
 ----
 
 . Use the `oc describe` command to inspect a Subscription resource:
 +
+[source,terminal]
 ----
 $ oc describe sub <subscription_name> -n <operator_namespace>
 ----
 
 . In the command output, find the `Conditions` section:
```

Comment:
Why do we want to do this? I'm not sure where the subscription info is in this output.

Reply:
@kalexand-rh thank you for your review and for highlighting this. I have updated this line with details about why a reader would want to do this.

Reply:
@jianzhangbjz can you please review the text that I have added to this line, from a QE perspective? It is as follows:

~~~
. In the command output, find the `Conditions` section for the status of Operator subscription condition types. In the following example, the `CatalogSourcesUnhealthy` condition type has a status of `false` because all available catalog sources are healthy:
~~~

Thanks!

Reply:
@pneedle-rh Thanks! Looks good to me.

Reply:
@jianzhangbjz thank you!